utilities/fault_injection_fs.cc

```diff
@@ -743,6 +791,26 @@ IOStatus FaultInjectionTestFS::DeleteFile(const std::string& f,
   return io_s;
 }
 
+IOStatus FaultInjectionTestFS::GetFileSize(const std::string& f,
+                                           const IOOptions& options,
+                                           uint64_t* file_size,
+                                           IODebugContext* dbg) {
+  // TODO: inject error, under what setting?
+  IOStatus io_s = target()->GetFileSize(f, options, file_size, dbg);
```

Comment:
> FaultInjectionTestFS historically did not emulate this behavior, only showing synced data from such read operations. (Different from FaultInjectionTestEnv--sigh.)

I am not sure about this statement yet. I thought `target()->GetFileSize()` here will eventually call down to PosixWritableFile::GetFileSize().